### PR TITLE
week-1 assignments

### PR DESCRIPTION
**Pull request by sachin286**
week 1 exercise 1.2 html page design

### CODE REVIEW

**Comment by sachin286** [comment]
@ShashankSuresh 
I have added both the assignments into the same PR

File: Week-1/Exercise-1.2/index.html [hunk 1]
```diff
@@ -0,0 +1,193 @@
+<html>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+
+<head>
+  <style>
+    * {
```
Comment (ShashankSuresh) [inline Week-1/Exercise-1.2/index.html]: Kindly create a new css file for styles.

File: Week-1/Exercise-1.2/index.html [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+      font-family: "Lato", sans-serif;
+      margin: 0;
+      list-style-type: none;
+    }
+
+    table {
```
Comment (ShashankSuresh) [inline Week-1/Exercise-1.2/index.html]: please add parent classes for all children and grandchildren.

File: Week-1/Exercise-1.1/Readme.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
+# How does a Browser work
```
Comment (ShashankSuresh) [inline Week-1/Exercise-1.1/Readme.md:1]: Good research and analysis. Good job. 

Can we talk a bit more on how the DOM is created? (HTML + CSS DOM) ?

**Review by ShashankSuresh — CHANGES_REQUESTED** [review]

**Comment by sachin286** [comment]
changes have been added to the same

File: Week-1/Exercise-1.2/index.html [hunk 4]
```diff
@@ -0,0 +1,99 @@
+<html>
+<head>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+</head>
+
+<body style="border-right:3px solid black; border-top: 3px solid black;">
```
Comment (ShashankSuresh) [inline Week-1/Exercise-1.2/index.html]: can we remove all the inline styles and move them to css classes?

**Comment by sachin286** [comment]
all the inline styles have been moved into the css file

File: Week-1/Exercise-1.2/style.css [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+  padding: 8px;
+}
+
+tr:nth-child(even) {
+  background-color: gainsboro;
+}
+
+.btn {
+  border: 1px solid black;
+  background-color: white;
```
Comment (ShashankSuresh) [inline Week-1/Exercise-1.2/style.css]: can we have this in css variables?

File: Week-1/Exercise-1.2/style.css [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+  color: blue;
+  font-weight: bold;
```
Comment (ShashankSuresh) [inline Week-1/Exercise-1.2/style.css]: can we have this in css variables?

File: Week-1/Exercise-1.2/style.css [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+}
+
+.bottom {
+  position: absolute;
+  bottom: 0;
+  /* set the bottom to 0*/
+  margin-left: 0px;
+  background-color: rgb(60, 90, 146);
```
Comment (ShashankSuresh) [inline Week-1/Exercise-1.2/style.css]: can we have this in css variables?

File: Week-1/Exercise-1.2/style.css [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+}
+
+.sidebar {
+  height: 81%;
+  width: 80x;
+  position: fixed;
+  z-index: 1;
+  top: 1px;
+  left: 0;
+  background-color: gainsboro;
```
Comment (ShashankSuresh) [inline Week-1/Exercise-1.2/style.css]: can we have this in css variables and as hex colors?

File: Week-1/Exercise-1.2/index.html [hunk 9]
```diff
@@ -0,0 +1,89 @@
+<html>
```
Comment (ShashankSuresh) [inline Week-1/Exercise-1.2/index.html:1]: Looks good. Good job :)

**Review by ShashankSuresh — CHANGES_REQUESTED** [review]

**Comment by sachin286** [comment]
all css variables added and updated the file

File: Week-1/Exercise-1.2/style.css [hunk 10]
```diff
@@ -0,0 +1,158 @@
+:root {
```
Comment (ShashankSuresh) [inline Week-1/Exercise-1.2/style.css:1]: Minor fix which can be done later. 

Rearrange based on variable type:

```
// Colors 
  --white: #ffffff;
  --black: #000000;
  --primary-blue: #1d0eed; // better naming convention
  --secondary-blue: #3c5a92; // better naming convention
  --gainsboro : #dcdcdc; // identify color using name

/* Font families. */
--font-family-sans: "Roboto", sans-serif;
--font-family-serif: "Playfair Display", serif;

/* Font sizes. */
--font-size-sm: 0.875rem;
--font-size-md: 1rem;
--font-size-lg: 1.125rem;
--font-size-xl: 1.25rem;
--font-size-2xl: 1.5rem;
--font-size-3xl: 2rem;
--font-size-4xl: 3rem;

/* Numeric keyword values */
font-weight: 100;
font-weight: 200;
font-weight: 300;
font-weight: 400;// normal
font-weight: 500;
font-weight: 600;
font-weight: 700;// bold
font-weight: 800;
font-weight: 900;
```

**Review by ShashankSuresh — APPROVED** [review]

**Pull request merged**